rep-0003.rst

```diff
@@ -180,6 +180,49 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2018)
+----------------------------------
+Required Support for:
+
+- Ubuntu Trusty (14.04)
+- Ubuntu Vivid (15.04)
+- Ubuntu Wily (15.10)
```

Comment:
k-turtle will not target trusty  and vivid but 16.04 according to http://wiki.ros.org/Distributions/ReleasePolicy